returnn/hdf.py

```diff
@@ -610,31 +620,42 @@ def _process_segment(segment: Tuple[str, float, float]):
                 if sys.byteorder == "big":
                     data_arr.byteswap()
                 data_bytes = data_arr.tobytes()
-                ffmpeg_out = sp.check_output(
-                    [
-                        "ffmpeg",
-                        "-hide_banner",
-                        "-loglevel",
-                        "error",
-                        "-y",
-                        "-ar",
-                        str(self.target_sampling_rate),
-                        "-f",
-                        "s16le",
-                        "-i",
-                        "-",
-                        "-c:a",
-                        c_codec,
-                        "-qscale:a",
-                        str(c_level),
-                        "-f",
-                        c_format,
-                        "-",
-                    ],
-                    bufsize=2**20,
-                    input=data_bytes,
-                    timeout=60,
-                )
+
+                try:
+                    ffmpeg_proc = sp.run(
+                        [
+                            "ffmpeg",
+                            "-hide_banner",
+                            "-loglevel",
+                            "error",
+                            "-y",
+                            "-ar",
+                            str(self.target_sampling_rate),
+                            "-f",
+                            "s16le",
+                            "-i",
+                            "-",
+                            "-c:a",
+                            c_codec,
+                            "-qscale:a",
+                            str(c_level),
+                            "-f",
+                            c_format,
+                            "-",
+                        ],
+                        bufsize=2**20,
+                        check=True,
+                        capture_output=True,
+                        input=data_bytes,
+                        timeout=60,
+                    )
+                    ffmpeg_out = ffmpeg_proc.stdout
+                except sp.ChildProcessError as e:
```

Comment:
Same question here. Why this? Why not directly output it?

Reply:
See other comment.